tests/h/api/search/index_test.py

```diff
@@ -111,6 +111,31 @@ def log(self, patch):
         return patch('h.api.search.index.log')
 
 
+@pytest.mark.usefixtures('BatchIndexer', 'BatchDeleter')
+class TestReindex(object):
+    def test_it_indexes_all_annotations(self, BatchIndexer):
+        session, es, request = mock.Mock(), mock.Mock(), mock.Mock()
```

Comment:
Yeah, I need to start using `mock.sentinel` more. Didn't do the assignment to a shorter variable though.